frontend/src/pages/SuperAdmin.js

```diff
@@ -1,8 +1,27 @@
 import React from 'react'
 import { useTranslation } from 'react-i18next'
 import { Route } from 'react-router-dom'
+import EditProductButton from '../components/listofproducts/EditProductButton'
 import UsersList from '../components/UsersList'
 import SuperAdminProductsPage from '../pages/SuperAdminProductsPage'
+import EditProductPage from './EditProductPage'
+
+const product = {
+  id: '628a5a49-8f5e-448f-8adf-158455cd98f8',
```

Comment:
Ha nincs még API, akkor mockoljunk axios-szal...

Reply:
Van BE hozzá, működik is, csupán amíg nincs terméklista, addig ezt használtam a validáláshoz.